OpenRA.Mods.Common/Activities/Air/HeliFly.cs

```diff
@@ -53,6 +53,9 @@ public override Activity Tick(Actor self)
 			if (IsCanceled || !target.IsValidFor(self))
 				return NextActivity;
 
+			if (helicopter.CenterPosition.Z == 0)
```

Comment:
#8866 introduced an `IsAtGroundLevel()` that works correctly for mods that use heightmaps.